File: UserAdmin/ClientApp/app/models/usuario.ts
import { MiembroCompensador } from "./miembro-compensador";
import { CuentaCompensacion } from "./cuenta-compensacion";

export interface Usuario {
  nombre: string;
  email: string;
  mc: string;
  cim: string;
  // mc: MiembroCompensador;
  // cim: CuentaCompensacion;
}

File: UserAdmin/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace UserAdmin.Models
{
    public partial class Usuario
    {
        public int UsuarioID { get; set; }
        public string Nombre { get; set; }
        public string Email { get; set; }
        public string MC { get; set; }
        public string CIM { get; set; }
    }
}
